static/app/views/relocation/getStarted.tsx

```diff
@@ -15,10 +15,13 @@ import {RelocationOnboardingContext} from 'sentry/views/relocation/relocationOnb
 import {StepProps} from './types';
 
 function GetStarted(props: StepProps) {
-  const regions = ConfigStore.get('regions');
   const [regionUrl, setRegionUrl] = useState('');
   const [orgSlugs, setOrgSlugs] = useState('');
   const relocationOnboardingContext = useContext(RelocationOnboardingContext);
+  const selectableRegions = ConfigStore.get('relocationConfig')?.selectableRegions;
```

Comment:
Should we default to empty array instead of undefined if `selectableRegions` is not defined?

Reply:
Done.